test/web-api.test.ts

```diff
@@ -678,6 +679,70 @@ describe('WebApi', () => {
     });
   });
 
+  describe('business_verification', () => {
+    it('successfully sends a business verification job', async () => {
+      const instance = new WebApi('001', 'https://a_callback.cb', Buffer.from(pair.public).toString('base64'), 0);
+      const partner_params = {
+        user_id: '1',
+        job_id: '1',
+        job_type: JOB_TYPE.BUSINESS_VERIFICATION,
+      };
+      const id_info = {
+        country: 'NG',
+        id_type: 'BUSINESS_REGISTRATION',
+        id_number: 'A000000',
+        business_type: 'co',
+      };
+
+      nock('https://testapi.smileidentity.com')
+        .post('/v1/business_verification', (body) => {
+          expect(body.partner_id).toEqual('001');
+          expect(body.country).toEqual(id_info.country);
+          expect(body.id_type).toEqual(id_info.id_type);
+          expect(body.id_number).toEqual(id_info.id_number);
+          expect(body.business_type).toEqual(id_info.business_type);
+          expect(body.partner_params.job_type).toEqual(JOB_TYPE.BUSINESS_VERIFICATION);
+          return true;
+        })
+        .reply(200, business_verification_response.success)
+        .isDone();
+
+      const resp = await instance.submit_job(partner_params, null, id_info, { signature: true });
+      expect(resp.data).toEqual(business_verification_response.success);
```

Comment:
Can you validate everything in the repsonse?

Reply:
This does validate everything in the response. The `businessVerificationResp` contains two response `{ success: {/*all response*/}, unsupported_business_type: {/*all response*/}`